cmd/cmd.go

```diff
@@ -38,6 +39,8 @@ const (
 	FlagLogFile = "log-file"
 	// FlagLogFormat is the name of log-format flag.
 	FlagLogFormat = "log-format"
+	// FlagLogRedact is whether to redact sensitive information in log.
+	FlagLogRedact = "log-redact"
```

Comment:
What does TiDB/TiKV name the configuration?

Reply:
https://github.com/pingcap/tidb/blob/master/sessionctx/variable/tidb_vars.go#L477
In TiDB it's a session variable `tidb_redact_log`.
https://github.com/tikv/tikv/blob/master/etc/config-template.toml#L846
In TiKV it's `redact-info-log` configuration in config file.